### DISCUSSION TITLE
Is in logtalk tabling possible?

### DISCUSSION THREAD

**Discussion by schwidom** [LogtalkDotOrg/logtalk3#205] [Q&A] [answered]
I see an example with tabling and the table keyword but there is nothing in the documentation. Is it safe for work?

**Comment by pmoura** [comment, marked as answer]
Logtalk tabling support, for the Prolog backends that implement tabling, is handled at the backend adapter files level. The read-only Logtalk `tabling` flag is set to `supported` in those cases. But note that portability may be an issue as tabling implementations differ between Prolog systems.
